benchmark/cuchembm/metrics/embedding.py

```diff
@@ -8,25 +8,27 @@
 logger = logging.getLogger(__name__)
 
 try:
-    import cuml # TODO is there a better way to check for RAPIDS?
-except:
+    import cupy as xpy
+    import cudf as xdf
+    from cuml.metrics import pairwise_distances, mean_squared_error
+    from cuml.linear_model import LinearRegression, ElasticNet
+    from cuml.svm import SVR
+    from cuml.ensemble import RandomForestRegressor
+    from cuchembm.metrics.utils import spearmanr
+    from cuchem.utils.distance import tanimoto_calculate
+    RAPIDS_AVAILABLE = True
+    logger.info('RAPIDS installation found. Using cupy and cudf where possible.')
+except ModuleNotFoundError as e:
     logger.info('RAPIDS installation not found. Numpy and pandas will be used instead.')
     import numpy as xpy
+    import pandas as xdf
```

Comment:
The one concern I've just thought of regarding leaving this code in is that users might assume sklearn etc is supported.